src/program/mod.rs

```diff
@@ -548,6 +552,12 @@ impl<'a> Program<'a> {
                     _ => {}
                 }
             }
+            if let Some(ts) = self.ui_state.rerender_time {
+                if Instant::now().duration_since(ts) > Duration::from_secs(2) {
```

Comment:
Seems similar to sdl2 timer system.
https://docs.rs/sdl2/0.32.2/sdl2/struct.TimerSubsystem.html#method.add_timer